src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -359,8 +359,11 @@ private bool IsFileSource()
             CancellationToken token)
         {
             var root = sourceUri.LocalPath;
-            var reader = new PackageArchiveReader(pathToPackage);
-            var packageIdentity = reader.GetIdentity();
+            PackageIdentity packageIdentity = null;
```

Comment:
Isn't this is a change that you already merged?

Reply:
sneaky 🐍

Reply:
It is, I just haven't rebased yet! It should get dropped from this on squash.